nativelib/src/main/resources/scala-native/delimcc/setjmp_amd64.S

```diff
@@ -1,5 +1,5 @@
 
-#if defined(__x86_64__) && (defined(__linux__) || defined(__APPLE__))
+#if defined(__x86_64__) && (defined(__linux__) || defined(__APPLE__) || defined(__FreeBSD__))
```

Comment:
I concur with this change.

I never tried running on 32 bit FreeBSD.  If the intent is
to support and & test such (which I doubt SN should be doing in the absence of a champion/advocate). does the
corresponding setjmp_amd32.s need a corresponding change?